apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -629,3 +646,14 @@ check_params(<<"channels.update.new">>, Params) when is_map(Params) ->
                                , mandatory => false})]);
 check_params(Method, Params) ->
     {error, {unknown, Method, Params}}.
+
+valid_error_code(ErrorCode) when is_integer(ErrorCode) ->
+    ErrorCode >= 1 andalso ErrorCode =< 16#ffFF;
```

Comment:
Why are you mixing upper and lower case?

Reply:
> Why are you mixing upper and lower case?

I find that it's easier to count the number of letters then. Perhaps overkill when it's only four of them. :)